assets/training/model_management/components/convert_model_to_mlflow/spec.yaml

```diff
@@ -28,6 +28,15 @@ inputs:
     description: Huggingface model id (https://huggingface.co/<model_id>). A required parameter for Huggingface model framework. Can be provided as input here or in model_download_metadata JSON file.
     optional: true
 
+  model_flavor:
+    type: string
+    enum:
+      - HFTransformersV2
```

Comment:
what about models that make use of pyfunc ex vision models. Could we also get approval from @rjaincc and team pls if they see any concern?

Reply:
This does not seem to be affecting vision flow. But it will be good to either change property name hf_mlflow_flavor or add pyfunc in the enum options as well.